lib/APPRemoteControl/src/App.cpp

```diff
@@ -390,16 +388,16 @@ void App_statusChannelCallback(const uint8_t* payload, const uint8_t payloadSize
 /**
  * Receives Turtle speed setpoints over SerialMuxProt channel.
  *
- * @param[in] payload       Linear and angular speed setpoints in a TurtleSpeed structure.
- * @param[in] payloadSize   Size of the TurtleSpeed structure.
+ * @param[in] payload       Linear and angular speed setpoints in a RobotSpeed structure.
+ * @param[in] payloadSize   Size of the RobotSpeed structure.
  * @param[in] userData      Instance of App class.
  */
 void App_turtleChannelCallback(const uint8_t* payload, const uint8_t payloadSize, void* userData)
```

Comment:
Change callback name.

Reply:
Your comment is not on the latest commit, where this change was already fixed